### PR TITLE
Complete is_hvac_sys_preheating_type_elec_resistance

### PR DESCRIPTION
**Pull request by weilixu**

### CODE REVIEW

**Review by yunjoonjung-PNNL — APPROVED** [review]
Looks great to me. Thank you for this effort!

File: rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/test_is_hvac_sys_preheating_type_elec_resistance.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            "id": "HW_Loop_1",
+            "type": "HEATING",
+        }
+    ],
+}
+
+TEST_RMD_FULL = {"id": "229_01", "ruleset_model_instances": [TEST_RMD]}
+
+
+def test__preheatheating_type_fluid_loop():
```
Comment (charliepnnl) [inline rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/test_is_hvac_sys_preheating_type_elec_resistance.py:56]: Shouldn't the test name match the function being tested?
Also, there should be an extra underscore between "..._type" and "fluid_loop".

File: rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/is_hvac_sys_preheating_type_elec_resistance.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        The HVAC system ID.
+
+    Returns
+    -------
+    bool
+        True: HVAC system preheating system has ELECTRIC_RESISTANCE as the heating type
+        False: HVAC system has a preheating system type other than ELECTRIC_RESISTANCE
+    """
+    # Get the hvac system
+    hvac_b = find_exact_one_hvac_system(rmi_b, hvac_b_id)
```
Comment (charliepnnl) [inline rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/is_hvac_sys_preheating_type_elec_resistance.py:28]: Shouldn't this be `_exactly_`?

**Review by charliepnnl — APPROVED** [review]
Approved with very small comments.

**Pull request merged**